modules/terraform/init.go

```diff
@@ -22,3 +22,17 @@ func InitE(t testing.TestingT, options *Options) (string, error) {
 	args = append(args, FormatTerraformPluginDirAsArgs(options.PluginDir)...)
 	return RunTerraformCommandE(t, options, args...)
 }
+
+// InitWithoutBackend calls terraform init and return stdout/stderr.
+func InitWithoutBackend(t testing.TestingT, options *Options) string {
```

Comment:
What is the use case for this?

Reply:
Since validation requires an initialized working directory, I tried to add the functionality to have an initialized working directory without accessing any configured remote backend. ( ref: https://www.terraform.io/docs/cli/commands/validate.html )
However that may be absolutely redundant, since the main goal of this PR is to add `terraform validate` option and the existing Init functions are enough for that.
I could revert the commit that adds this functionality if you have concerns about it :)

Reply:
commit with this change reverted :)